python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
-   in ascending order, which gives the sample
-   :math:`\left\{ y^{(1)},\ldots,y^{(N)} \right\}`,
+We assume that we have a sample of size :math:`\sampleSize` of :math:`X`, denoted by :math:`(x_1, \dots, x_\sampleSize)`.
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`\sampleSize`, we seek for the smallest :math:`k` such that:
+
+.. math::
+    :label: EqOrderStat
 
--  this empirical estimation of the quantile is then calculated by the
-   formula:
+    \Prob{X_{(k)} \geq x_{\alpha}} \geq \beta
 
-   .. math::
+The order statistics :math:`X_{(k)}` follows the distribution which probability density function and cumulated distribution function
+are defined by:
 
-      \begin{aligned}
-            \widehat{q}_{y^i}(\alpha) = y^{([N\alpha]+1)}
-          \end{aligned}
+.. math::
+    :label: DistOrderStat
 
-where :math:`[N\alpha]` denotes the integral part of
-:math:`N\alpha`.
+    F_{X_{(k)}}(x) & = \sum_{i=k}^{\sampleSize} \binom{\sampleSize}{i}\left[F(x)\right]^i \left[1-F(x)
+    \right]^{\sampleSize-i} \\
+    p_{X_{(k)}}(x) & = (\sampleSize-k+1)\binom{\sampleSize}{k-1}\left[F(x)\right]^{k-1} \left[1-F(x)
+    \right]^{\sampleSize-k} p(x)
 
-For example, if :math:`N=100` and :math:`\alpha = 0.95`,
-:math:`\widehat{q}_Z(0.95)` is equal to :math:`y^{(96)}`, which is the
-:math:`5^\textrm{th}` largest value of the sample
-:math:`\left\{ y^i_1,\ldots,y^i_N \right\}`. We note that this
-estimation has no meaning unless :math:`1/N \leq \alpha \leq 1-1/N`. For
-example, if :math:`N=100`, one can only consider values of a to be
-between 1% and 99%.
+We notice that :math:`F_{X_{(k)}}(x) = \overline{F}_{(\sampleSize,F(x))}(k-1)` where :math:`F_{(\sampleSize,F(x))}` is the cumulated
+distribution function of the Binomial distribution :math:`\cB(\sampleSize,F(x))`.
 
-It is also possible to calculate an upper limit for the quantile with a
-confidence level :math:`\beta` chosen by the user; one can then be sure
-with a :math:`\beta` level of confidence that the real value of
-:math:`q_{Y^i}(\alpha))` is less than or equal to
-:math:`\widehat{q}_{Y^i}(\alpha)_{\sup}`:
+Then, we have:
 
 .. math::
 
-   \begin{aligned}
-       \Prob{q_{Y^i}(\alpha) \leq \widehat{q}_{Y^i}(\alpha)_{\sup}} = \beta
-     \end{aligned}
+    F_{X_{(k)}}(x_{\alpha}) = \sum_{i=k}^{\sampleSize} \binom{\sampleSize}{i} \alpha^i (1-\alpha)^{\sampleSize-i}
+    = \overline{F}_{(\sampleSize,\alpha)}(k-1)
 
-The most robust method for calculating this upper limit consists of
-taking
-:math:`\widehat{q}_{Y^i}(\alpha)_{\sup} = y^{(j(\alpha,\beta,N))}` where
-:math:`j(\alpha,\beta,N)` is an integer between 2 and :math:`N` found by
-solving the equation:
+and relation :eq:`EqOrderStat` can be written as:
 
 .. math::
+    :label: EqOrderStat2
 
-   \begin{aligned}
-       \sum_{k=1}^{j(\alpha,\beta,N) - 1} C^k_N \alpha^k \left( 1-\alpha \right)^{N-k} = \beta
-     \end{aligned}
+    1-F_{X_{(k)}}(x_{\alpha})\geq \beta \\
+    F_{\sampleSize, \alpha}(k-1)\geq \beta
```

Comment:
These are two different equation, where the second is implied by the first one.
```suggestion
    1 - F_{X_{(k)}}(x_{\alpha}) \geq \beta

This implies:

.. math::
    F_{\sampleSize, \alpha}(k - 1) \geq \beta
```

Reply:
ok